x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -399,6 +404,35 @@ class AgentPolicyService {
     return agentPolicy;
   }
 
+  private async populateRootIntegrations(
+    soClient: SavedObjectsClientContract,
+    agentPolicy: AgentPolicy
+  ) {
+    if (!agentPolicy.package_policies) {
+      return;
+    }
+    const packageInfos = await pMap(
+      agentPolicy.package_policies,
+      async (packagePolicy) => {
+        if (!packagePolicy.package) return null;
+        return await getPackageInfo({
+          savedObjectsClient: soClient,
+          pkgName: packagePolicy.package.name,
+          pkgVersion: packagePolicy.package.version,
+          prerelease: true,
+        });
+      },
+      { concurrency: 5 }
+    );
+
+    agentPolicy.rootIntegrations = packageInfos
```

Comment:
`/agent_policies` API returns the packages requiring root from the list of package policies:
```
rootIntegrations: [{name: "auditd_manager", title: "Auditd Manager"}]
```